### PR TITLE
Add additional metrics for deduped deployments and alerts

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

Title

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

CI

### CODE REVIEW

File: central/sensor/service/pipeline/alerts/pipeline.go [hunk 1]
```diff
@@ -72,6 +73,10 @@ func (s *pipelineImpl) Run(ctx context.Context, clusterID string, msg *central.M
 		return nil
 	}
 
+	for _, alert := range alertResults.GetAlerts() {
+		pipelineMetrics.IncTotalAlertsReceived(msg.GetEvent().GetAction(), alertResults.GetStage(), alert.GetPolicy().GetName())
```
Comment (theencee) [inline central/sensor/service/pipeline/alerts/pipeline.go:77]: Same thing about policy name

File: central/detection/alertmanager/alert_manager_impl.go [hunk 2]
```diff
@@ -317,6 +318,8 @@ func (d *alertManagerImpl) mergeManyAlerts(
 			mergedAlert := mergeAlerts(matchingOld, alert)
 			if mergedAlert != matchingOld && !proto.Equal(mergedAlert, matchingOld) {
 				updatedAlerts = append(updatedAlerts, mergedAlert)
+			} else {
+				metrics.IncDedupedAlerts(alert.GetLifecycleStage(), alert.GetPolicy().GetName())
```
Comment (theencee) [inline central/detection/alertmanager/alert_manager_impl.go:322]: I'm not super familiar with prometheus and how it handles arbitrarily large tags, but policy name can be long. What about id? Can always correlate later by pulling policies from store/db.

**Review by theencee — APPROVED** [review]

File: central/detection/alertmanager/alert_manager_impl.go (hunk 2 above)
Comment (connorgorman) [reply central/detection/alertmanager/alert_manager_impl.go:322]: My concern is if someone creates a new policy, then we won't know the content

**Pull request closed without merging**